### ISSUE TITLE
Unresolved reference: registerImagePicker

### ISSUE THREAD

**Issue by HosseinKurd** [esafirm/android-image-picker#348]
Unresolved reference: registerImagePicker in Versions **2.4.5 , 3.0.0-alpha1 , v3.0.0-alpha2**

can not call:

```
 val launcher = registerImagePicker {result: List<Image> ->
            result.forEach { image ->
                println(image)
            }
        }
        launcher.launch()
```

**Comment by coldfish** [comment]
I think you're trying to use 3.x.x versions. Please use this Gradle link using Jitpack:

`implementation 'com.github.esafirm:android-image-picker:3.0.0-alpha2'`

instead of 2.x.x:

`implementation 'com.github.esafirm.android-image-picker:imagepicker:2.4.5'`

**Comment by HosseinKurd** [comment]
> I think you're trying to use 3.x.x versions. Please use this Gradle link using Jitpack:
> 
> `implementation 'com.github.esafirm:android-image-picker:3.0.0-alpha2'`
> 
> instead of 2.x.x:
> 
> `implementation 'com.github.esafirm.android-image-picker:imagepicker:2.4.5'`

Version `3.0.0-alpha2` , `3.0.0-alpha1` and `2.4.5` Not works for me

**Comment by coldfish** [comment]
Are you sure if you've provisioned the jetpack.io in build.gradle? 

```
allprojects {
    repositories {
        maven { url "https://jitpack.io" }
    }
}
```

**Comment by HosseinKurd** [comment]
> Are you sure if you've provisioned the jetpack.io in build.gradle?
> 
> ```
> allprojects {
>     repositories {
>         maven { url "https://jitpack.io" }
>     }
> }
> 
jitpack has been set but not works

**Comment by coldfish** [comment]
Can you zip a sample project and share via cloud link?

**Comment by esafirm** [comment]
@HosseinKurd can you get the sample app working?

**Comment by esafirm** [comment]
Just updated the readme, it turns out I put the wrong artifact on it because we used to have RxJava module.
https://github.com/esafirm/android-image-picker/issues/348#issuecomment-821970448 should works.

**Comment by kenilt** [comment]
I have faced the same problem, then I found the reason why. My project use Dagger, then I extend all my activity to `DaggerAppCompatActivity` which was a successor of `androidx.core.app.ComponentActivity`. However, this library is writing the extension for `androidx.activity.ComponentActivity`, which is extended to `androidx.core.app.ComponentActivity`. So they are not matched together, so it was not resolved in my project.
